.mcp.json

```diff
@@ -0,0 +1,20 @@
+{
+	"mcpServers": {
+		"go-language-server": {
+			"type": "stdio",
+			"command": "go",
+			"args": [
+				"run",
+				"github.com/isaacphi/mcp-language-server@latest",
+				"-workspace",
+				"./",
+				"-lsp",
+				"go",
+				"--",
+				"run",
+				"golang.org/x/tools/gopls@latest"
+			],
```

Comment:
😍

Reply:
Yeah, it doesn't always use it, but when it does, it's almost like a cheat/hack. 😅